.java-version

```diff
@@ -0,0 +1 @@
+11.0
```

Comment:
What's this?

Reply:
Sorry, it is a file added by standard macos tooling (`jenv`) to manage jdk by directory. I'll remove it.